### PR TITLE
Create README_CN.md

### PR DESCRIPTION
**Pull request by rexsama**

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+**[许可证](#licence)** |
+**[相关资源](#resources)** |  
+
+# [Nebula AI Worker](https://github.com/nebulaai/ai-worker)
+
+Orion AI Worker 官方软件发布.
+
+## AI Worker 版本 V2
+
+**AI Workers** 是 [Orion 云端平台](https://orioncloud.io/)中的资源提供者. 他们执行 AI 任务并获得 NBAI 币作为奖励.
```
Comment (NBAI-Ray) [inline README_CN.md]: orion is 云计算平台

Comment (NBAI-Cloud-Platform) [inline README_CN.md]: orion is 云计算平台

**Review by NBAI-Cloud-Platform — CHANGES_REQUESTED** [review]
Please fix the translaction for 'orion cloud platform' mentioned in the comment

File: README_CN.md (hunk 1 above)
Comment (rexsama) [reply README_CN.md]: 更改完成。

**Pull request merged**